### PR TITLE
feat: add `skipSynchronizationBeforeHeight` unsafe option

### PR DESCRIPTION
**Pull request by jawid-h**
## Issue being fixed or feature implemented

Adding an unsafe option to skip number of blocks on initial sync as per #216 

## What was done?

- added `unsafeOptions` namespace
- added `skipSynchronizationBeforeHeight` option

## How Has This Been Tested?

Unit tests.

## Breaking Changes

None.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: src/types/Wallet/Wallet.d.ts [hunk 1]
```diff
@@ -59,5 +59,10 @@ export declare namespace Wallet {
         privateKey?: PrivateKey | string;
         HDPrivateKey?: HDPrivateKey | string;
         HDPublicKey?: HDPublicKey | string;
+        debugOptions?: IWalletDebugOptions;
```
Comment (shumkov) [inline src/types/Wallet/Wallet.d.ts:62]: What is the point to introduce this namespace?

Comment (jawid-h) [reply src/types/Wallet/Wallet.d.ts:62]: Why not?

Comment (jawid-h) [reply src/types/Wallet/Wallet.d.ts:62]: I was trying to follow this issue https://github.com/dashevo/wallet-lib/issues/216

Comment (shumkov) [reply src/types/Wallet/Wallet.d.ts:62]: debugOptions sounds confusing honestly. What if we rename this namespace to `unsafe`? Or rename the option to `unsafeSkipSynchronizationBeforeHeight`? @antouhou

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**